src/com/hotmail/kalebmarc/textfighter/player/Settings.java

```diff
@@ -9,7 +9,8 @@
 
     public static boolean difLocked = false;
     private static String difficulty;
-    private static boolean godMode = false;
+    private static boolean godMode = false;interest

```

Comment:
I believe this wasn't meant to be included
```suggestion
    private static boolean godMode = false;

```